### PR TITLE
Added image to text project

### PR DESCRIPTION
**Pull request by Utsavladia**
Closes: #442  

- **Title** :  Image to text project
- **Name:** : Utsav Ladia
- **Idenitfy yourself:** :  GSSOC

### Screenshots 📷
image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Avdhesh-Varshney** [comment]
Can you review your pr? And where is your project?

**Comment by Utsavladia** [comment]
@Avdhesh-Varshney 
I fixed the project section previously it got added as submodule

**Comment by Utsavladia** [comment]
@Avdhesh-Varshney 
Is this alright to be merged?

**Comment by Utsavladia** [comment]
@Avdhesh-Varshney kindly reivew this gssoc is getting ended

**Comment by Utsavladia** [comment]
@Avdhesh-Varshney 
Kindly merge this PR before closing of GSSOC 🙂

File: Next-JS-Projects/Intermediate/text-to-image/README.md
Comment (Avdhesh-Varshney) [inline Next-JS-Projects/Intermediate/text-to-image/README.md:1]: - Follow the readme template.
- Add ss file with provide file conventions in guidelines.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Carefully follow the guidelines of project is, `Text-To-Image` ok.

File: Next-JS-Projects/Intermediate/text-to-image/README.md
Comment (Avdhesh-Varshney) [reply Next-JS-Projects/Intermediate/text-to-image/README.md:1]: Also enlist your project in Next-JS-Projects readme file.

**Pull request closed without merging**